packages/fx-core/src/plugins/resource/cicd/result.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { FxError, Result, SystemError, UserError, err, ok } from "@microsoft/teamsfx-api";
+
+import { Alias } from "./constants";
+
+export type FxResult = Result<any, FxError>;
```

Comment:
Do we need to make FxResult as ```Result<boolean, FxError>```? Since we use ```  if (scaffolded.isOk() && !scaffolded.value)``` to try to get boolean value from scaffolded.value?

Reply:
Unnecessary, dev test is passed and scaffolded.value will not always be true. To give further reason, Result type is generic type when creating obj by ok(true), the generic type is fixed to boolean.